crates/re_viewer/src/app.rs

```diff
@@ -1320,6 +1320,22 @@ impl eframe::App for App {
                 .add(egui_ctx.input(|i| i.time), seconds);
         }
 
+        #[cfg(target_arch = "wasm32")]
```

Comment:
actually why not also no native 🤔 
although tbf I would have expect those buttons to change selection in the selection history